src/ambianic/pipeline/store.py

```diff
@@ -91,22 +91,40 @@ def _save_sample(self,
         inf_json = []
         if inference_result:
             for inf in inference_result:
-                label, confidence, box = inf[0:3]
-                log.info('label: %s , confidence: %.0f, box: %s',
-                        label,
-                        confidence,
-                        box)
-                one_inf = {
-                    'label': label,
-                    'confidence': float(confidence),
-                    'box': {
-                        'xmin': float(box[0]),
-                        'ymin': float(box[1]),
-                        'xmax': float(box[2]),
-                        'ymax': float(box[3]),
+                label, confidence = inf[0:2]
+
+                if isinstance(inf[2], (list, tuple)):
```

Comment:
It think it's time to introduce more structure and easier to maintain logic. Remember the earlier issue to refactor TFImageDetection? 

store.py needs to use a more intuitive logic for saving various detections. Maybe we pull the code for generating a serializable event structure in the detection classes and store simply uses that and dumps it to the event log. 

As we are adding more detections, it becomes unclear why store needs to know the internal data structure details of each detection output.

Reply:
@bhavikapanara did you see this comment? It is still valid. The way we feed a bounding box to the store class should be pulled out of here and into the model output since each model has its own way of defining a bounding box.